src/main/java/com/google/netpcapanalysis/models/ReverseDNS.java

```diff
@@ -0,0 +1,16 @@
+package com.google.netpcapanalysis.models;
+
+public class ReverseDNS {
+  public boolean authority;
+  public boolean server;
+  public String record;
+
+  public ReverseDNS() {
+    authority = false;
+    server = false;
+  }
+
+  public ReverseDNS(String record, boolean server, boolean authority) {
+
```

Comment:
did you forget to add the implementation? or you dont need this overloaded constructor?

Reply:
I went ahead and filled it out